spec/models/email_spec.rb

```diff
@@ -31,19 +31,11 @@
     }
 
     it "sets subject" do
-      expect(email.subject).to eq("Title")
+      expect(email.subject).to_not be_nil
```

Comment:
Why are we only testing that these are not `nil` now?

Reply:
I figured since we already have tests for the `EmailRenderer`, it didn't seem useful to check the contents again.

Reply:
Could we stub the methods on `EmailRenderer` and then test that they are set to what the `EmailRenderer` returns instead?

Reply:
Sure, fixed that now!